### DISCUSSION TITLE
Actividad 0 - Llontop, Jose

### DISCUSSION THREAD

**Discussion by JoseLlontop** [frlp-utn-ingsoft/catedra#30] [Show and tell]
https://github.com/JoseLlontop/actividad-jose-llontop.git

**Comment by RodrigoJacznik** [comment]
Pudiste votar en la encuesta que esta en la db?

**Reply by JoseLlontop** [reply]
Buenas, me olvide de agregar las opciones a la encuesta, recién lo corregí

**Reply by RodrigoJacznik** [reply]
Genial, Lo pude probar y funciona correctamente.
Los commits y sus mensajes están correctos
La actividad esta aprobado.
